### DISCUSSION TITLE
Use without Calibre?

### DISCUSSION THREAD

**Discussion by Shohreh** [noDRM/DeDRM_tools#448] [Q&A] [answered]
Hello,

Unless I missed it, this question isn't mentioned in the FAQ, and I have found no answer in the archives.

I only use Calibre to remove DRMs from books I get from the library before copying EPUB files onto my e-reader. That's a big piece of software for just that purpose.

Can this tool be used on its own, or must I keep using Calibre?

Thank you.

**Comment by ElleKayEm** [comment, marked as answer]
There were stand alone Mac and Windows apps.  But support for them was dropped in favor of just maintaining the plugin many versions ago.
